app/src/main/java/org/simple/clinic/user/LoggedInUserHttpInterceptor.kt

```diff
@@ -9,12 +9,13 @@ import javax.inject.Inject
 class LoggedInUserHttpInterceptor @Inject constructor() : Interceptor {
 
   // Ugly hack to avoid a cyclic dependency between this class and UserSession.
-  private val userSession by lazy { ClinicApp.appComponent.userSession() }
+  private val userSession
+    get() = ClinicApp.appComponent.userSession()
```

Comment:
I changed this to a getter because lazy initialization is unnecessary. Dagger will handle it for us.

Reply:
Looks better 👍